extensions-contrib/graphite-emitter/src/main/java/io/druid/emitter/graphite/GraphiteEmitter.java

```diff
@@ -175,11 +173,12 @@ public void run()
             if (e instanceof InterruptedException) {
               Thread.currentThread().interrupt();
             } else if (e instanceof SocketException) {
+              pickledGraphite.close();
+              log.info("had exception [%s] trying to re-connect to graphite server", e.getMessage());
```

Comment:
The exception is already logged a few lines above, maybe this logging statement should just tell "trying to re-connect.."?